eclipse-scout-core/src/calendar/Calendar.ts

```diff
@@ -229,11 +209,28 @@ export class Calendar extends Widget implements CalendarModel {
     return new KeyStrokeContext();
   }
 
+  protected _createDefaultCalendar(): CalendarDescriptor {
+    return {
+      calendarId: UuidPool.ZERO_UUID,
+      selectable: true,
+      visible: true
+    };
+  }
+
+  override init(model: InitModelOf<this>) {
+    super.init(model);
+  }
+
   protected override _init(model: InitModelOf<this>) {
     super._init(model);
-    this.yearPanel = scout.create(YearPanel, {
+    this.calendarSidebar = scout.create(CalendarSidebar, {
       parent: this
+
     });
+    this.yearPanel = this.calendarSidebar.yearPanel;
+    this.calendarsPanel = this.calendarSidebar.calendarsPanel;
+
+    this.calendarsPanel.treeBox.on('propertyChange:value', this._onCalendarTreeBoxValueChanged.bind(this));
```

Comment:
Rename to _onCalendarVisibilityChanged

Reply:
since you listen to 'propertyChange:value' event you should follow the naming pattern _onXyzValueChange, where Xyz should be the object to which the listener is added